platform/feature-api/communication/postoffice/src/types.ts

```diff
@@ -1,52 +1,52 @@
 /**
- * This module defines the types used for specifying endpoints.
+ * This module defines the types used for specifying backend endpoints.
  *
- * An _endpoint_ is an API that can be implemented by a server and called by a
- * client. Endpoints are structured as objects with methods; methods may return
+ * A _backend endpoint_ is an API that can be implemented by a server and called by a
+ * client. Backend endpoints are structured as objects with methods; methods may return
  * either values or a nested endpoint.
  *
  * An _endpoint specification_ is a mere type that is never used directly. See
- * [[EndpointSpec]] for details.
+ * [[BackendSpec]] for details.
  *
  * @packageDocumentation
  */
 
 /**
- * Interface denoting a value endpoint of type `T`. This interface is purely
+ * Interface denoting a value backend endpoint of type `T`. This interface is purely
  * virtual and no instances are generated.
  */
-export interface ValueEndpointSpec<T> {
+export interface ValueBackendSpec<T> {
     endpointType: "value";
     value: T;
 }
 
 /**
- * Interface denoting a method endpoint of type `T`. This interface is purely
+ * Interface denoting a method backend endpoint of type `T`. This interface is purely
  * virtual and no instances are generated.
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export interface ObjectEndpointSpec<T extends Record<string, (...args: any[]) => EndpointSpec>> {
+export interface ObjectBackendSpec<T extends Record<string, (...args: any[]) => BackendSpec>> {
     endpointType: "object";
```

Comment:
Not clear why we need this instance variable. Doesn't type introspection give you the same information?